libs/feature/search/src/lib/favorites/favorite-star/favorite-star.component.ts

```diff
@@ -74,10 +75,34 @@ export class FavoriteStarComponent implements AfterViewInit, OnDestroy {
         })
       }
     })
+    this.countSubscription = this.favoritesService.myFavoritesUuid$
```

Comment:
Not 100% sure why, but changing it to `modifiedFavorites$` makes it come back to the old behavior and doesn't even update the count on click (on the same page). Also, changing it to `modifiedFavorites$` only means that in the case the count is already X when the user adds it to its favorites, the display count will display '1' since the value of `modifiedFavorites$` is never synchronized with the api response in the first place and starts as an empty array. It also means that when the user adds its first favorite, `modifiedFavorites$` will only contain this record and all other favorites will display a count of '0'. I hope it's kind of clear?

Reply:
> Not 100% sure why, but changing it to `modifiedFavorites$` makes it come back to the old behavior and doesn't even update the count on click (on the same page). Also, changing it to `modifiedFavorites$` only means that in the case the count is already X when the user adds it to its favorites, the display count will display '1' since the value of `modifiedFavorites$` is never synchronized with the api response in the first place and starts as an empty array. It also means that when the user adds its first favorite, `modifiedFavorites$` will only contain this record and all other favorites will display a count of '0'. Let me know if it's not clear :)